rllib/agents/tests/test_trainer.py

```diff
@@ -33,16 +33,21 @@ def test_validate_config_idempotent(self):
         """
         # Given:
         standard_config = copy.deepcopy(COMMON_CONFIG)
+        trainer = pg.PGTrainer(env="CartPole-v0", config=standard_config)
```

Comment:
this doesn't seem to be belong to this PR?

Reply:
It does, we changed the `validate_config` method from `static` to normal, so now we do require an actual Trainer object to call this method, which we do here.

Reply:
Why would changing it from static to normal require that the trainer call this with the standard config?

Reply:
Sorry, I think I explained in a misleading way: Not the standard config is the problem/important here. It could be any config for this test that is a ok PG config!
But you do have to call it on an instantiated object, not just call the static function like before.